lib/css/base/_stacks-body.less

```diff
@@ -33,10 +32,9 @@ body {
 }, @force-selector: true);
 
 body {
-    #stacks-internals #load-config();
     box-sizing: border-box;
     min-height: 100%;
-    background-color: @body-background;
+    background-color: var(--white);
```

Comment:
I'd recommend making this a `theme` variable. We use `--theme-background-color` in Core, so shadowing this would allow us to drop some legacy code there.